packages/runtime/src/next/components/tests/controls/__snapshots__/checkbox-control.test.tsx.snap

```diff
@@ -4,8 +4,10 @@ exports[`Page Checkbox control data v0 when value is false: resolvedValue 1`] =
 
 exports[`Page Checkbox control data v0 when value is false: snapshot 1`] = `
 {
-  "apiOrigin": "https://test-api-origin.com",
-  "cacheData": {},
+  "cacheData": {
+    "apiResources": {},
+    "localizedResourcesMap": {},
+  },
```

Comment:
Here and below, lots of Jset snapshot updates reflecting changes to Makeswift snapshot structure.